test/unit/ngsiv2/examples/contextRequests/updateContextCommandStatus.json

```diff
@@ -0,0 +1,17 @@
+{
+  "contextElements": [
```

Comment:
This seems to be an update in NGSIv1 syntax, inside ngsiv2/examples directory... A bit weird. Maybe I'm missing something?

Reply:
Very good point, part of the functionality related to commands was not correct. :(

Reply:
Fixed with 746b6cd05e734fbf06afde2065ae5a32903f6b71